<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="AKHIL-TS" name="com.localassessment.Test_Suite_Print_Login_Codes" tests="10" failures="1" timestamp="10 Oct 2018 08:37:17 GMT" time="99.191" errors="0">
  <testcase name="TCED13307" time="22.866" classname="com.localassessment.Test_Suite_Print_Login_Codes"/>
  <testcase name="TCED13304" time="1.378" classname="com.localassessment.Test_Suite_Print_Login_Codes">
    <failure type="java.lang.AssertionError" message="failed to assert text Select School   expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: failed to assert text Select School   expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.localassessment.Test_Suite_Print_Login_Codes.TCED13304(Test_Suite_Print_Login_Codes.java:132)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- TCED13304 -->
  <testcase name="TCED13306" time="22.878" classname="com.localassessment.Test_Suite_Print_Login_Codes"/>
  <testcase name="TCED13310" time="1.273" classname="com.localassessment.Test_Suite_Print_Login_Codes"/>
  <testcase name="TCED13301" time="6.006" classname="com.localassessment.Test_Suite_Print_Login_Codes"/>
  <testcase name="TCED13309" time="12.151" classname="com.localassessment.Test_Suite_Print_Login_Codes"/>
  <testcase name="TCED13303" time="0.510" classname="com.localassessment.Test_Suite_Print_Login_Codes"/>
  <testcase name="TCED13305" time="4.176" classname="com.localassessment.Test_Suite_Print_Login_Codes"/>
  <testcase name="TCED13302" time="4.794" classname="com.localassessment.Test_Suite_Print_Login_Codes"/>
  <testcase name="TCED13308" time="23.159" classname="com.localassessment.Test_Suite_Print_Login_Codes"/>
</testsuite> <!-- com.localassessment.Test_Suite_Print_Login_Codes -->
